bpf/tests/nat_test.h

```diff
@@ -99,3 +101,52 @@ void test_snat_v4_track_local() {
     assert(!snat_v4_track_local(&ctx, &tuple, &state, NAT_DIR_EGRESS, 0,
                                 &target));
 }
+
+// Create a fake map.
+HASHMAP(struct ipv4_ct_tuple, struct ipv4_nat_entry) ipv4_ct_tuple_map;
+
+// Define the compare_func
+int bpf_compare(const void *a, const void *b)
+{
+    return __bpf_memcmp_builtin(a, b, sizeof(struct ipv4_ct_tuple));
+}
+
+// Define the hash_func
+size_t bpf_hash(const void *key)
+{
+    return hashmap_hash_default(key, sizeof(struct ipv4_ct_tuple));
+}
+
+// Wrap the fake map operations into callbacks.
```

Comment:
Can you describe in the comment that the "map" argument is actually not used, but the map itself is encoded in the function? (so we need different callback funcs for different maps).

Reply:
Added.